src/autotester/server/utils/tests/path_management_test.py

```diff
@@ -0,0 +1,35 @@
+from autotester.server.utils.path_management import current_directory, add_path
+import tempfile
+import os
+import sys
+
+
+def test_current_directory():
+    dir = tempfile.gettempdir()
+    curr_dir = os.getcwd()
+    with current_directory(dir):
+        temp_dir = os.getcwd()
+    assert temp_dir == dir
```

Comment:
Please make all of the assert statements into their own tests. Use fixtures for shared setup if needed.